RubeesEat.IntegrationTests/Controllers/BillControllerTests.cs

```diff
@@ -18,6 +18,7 @@ public class BillControllerTest : IntegrationTestBase
         var content = new FormUrlEncodedContent(new[]
         {
             new KeyValuePair<string, string>("billDescription", "test234"),
+            new KeyValuePair<string, string>("billDate", DateTime.Today.ToString("yyyy-MM-dd")),
```

Comment:
We should use a fixed date here and assert that this date was actually used when creating the ticket. Especially with dates it is easy to default to the current date, which is not want we want here.